vagrant/deploy.sh

```diff
@@ -17,6 +17,7 @@ vagrant up
 export VAGRANT_IP=$(vagrant ssh -- curl -qs http://169.254.169.254/latest/meta-data/public-ipv4)
 export CONCOURSE_URL=http://localhost:8080
 export FLY_TARGET=${DEPLOY_ENV}-bootstrap
+export ACCOUNT=${ACCOUNT:-development}
```

Comment:
I wonder whether it would be better to call this env var `AWS_ACCOUNT` to avoid any confusion about which account we're referring to.